packages/style-minify/README.md

```diff
@@ -0,0 +1,100 @@
+# Neutrino Style Minify Middleware
```

Comment:
Hi! Thank you for the PR :-)
Could you copy this README to docs/ too? (I'm hoping we can do away with this duplication soon)

Reply:
of course, forgot!